### PR TITLE
Refactor Tag class to Parcel class

### LINKED ISSUE #47: Add Parcels
As a parcel delivery rider (basic), I can check what specific parcels I need to deliver to a specific address so that I will delivery the correct parcel to the correct customers.

### LINKED ISSUE #51: Implement adding parcels to delivery
A delivery/person can have multiple parcels

### PR DESCRIPTION
**Pull request by joshua-tyh**
The new Parcel class is identical to the previous Tag class, that is a Parcel currently has the same fields and methods as the previous Tag class.

All previous logic is preserved throughout the codebase.

Updated JUnit Test files accordingly and passed all test cases.

closes #47 & closes #51

### CODE REVIEW

File: src/main/java/bookopedia/model/person/Person.java [hunk 1]
```diff
@@ -23,32 +23,32 @@ public class Person {
...
-    private final Set<Tag> tags = new HashSet<>();
+    private final Set<Parcel> parcels = new HashSet<>();
     private final DeliveryStatus deliveryStatus;
 
     /**
      * Every field must be present and not null except phone and email.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags, DeliveryStatus deliveryStatus) {
-        requireAllNonNull(name, address, tags);
+    public Person(Name name, Phone phone, Email email, Address address, Set<Parcel> parcels, DeliveryStatus deliveryStatus) {
```
Comment (Mehvin) [inline src/main/java/bookopedia/model/person/Person.java:32]: Line length longer than 120 characters here

File: src/main/java/bookopedia/storage/JsonAdaptedPerson.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
 import bookopedia.model.person.Name;
 import bookopedia.model.person.Person;
 import bookopedia.model.person.Phone;
-import bookopedia.model.tag.Tag;
+import bookopedia.model.parcel.Parcel;
```
Comment (Mehvin) [inline src/main/java/bookopedia/storage/JsonAdaptedPerson.java:19]: Checkstyle lexicographical error here

**Review by Mehvin — CHANGES_REQUESTED** [review]
Some minor checkstyle errors

File: src/main/java/bookopedia/logic/commands/AddCommand.java [hunk 3]
```diff
@@ -20,14 +20,14 @@ public class AddCommand extends Command {
             + CliSyntax.PREFIX_PHONE + "PHONE "
             + CliSyntax.PREFIX_EMAIL + "EMAIL "
             + CliSyntax.PREFIX_ADDRESS + "ADDRESS "
-            + "[" + CliSyntax.PREFIX_TAG + "TAG]...\n"
+            + "[" + CliSyntax.PREFIX_PARCEL + "TAG]...\n"
```
Comment (shazxrin) [inline src/main/java/bookopedia/logic/commands/AddCommand.java:23]:
```suggestion
            + "[" + CliSyntax.PREFIX_PARCEL + "PARCEL]...\n"
```

**Review by Mehvin — APPROVED** [review]

**Pull request merged**